tests/app/main/test_briefs.py

```diff
@@ -2297,3 +2332,20 @@ def test_supplier_details_shown(self, data_api_client):
         assert requirements_data.row(1).cell(1) == "02/02/2017"
         assert requirements_data.row(2).cell(0) == "Email address"
         assert requirements_data.row(2).cell(1) == "contact@big.com"
+
+    def test_visit_to_start_page_redirects_to_result_page(self, data_api_client):
```

Comment:
Your other option would be that instead of visiting the create page, you just mock the flash message and visit the result page and assert that the correct banner was shown. However, I think this will do for the moment